packages/ZeroBounce.dpk

```diff
@@ -25,7 +25,7 @@ package ZeroBounce;
 {$IMAGEBASE $400000}
 {$DEFINE DEBUG}
 {$ENDIF IMPLICITBUILDING}
-{$LIBVERSION '1.0.0'}
+{$LIBVERSION '1.0.1'}
```

Comment:
if we add a new feature it can be 1.1.0 (I'll admit that this feels like a small update to me too)

Reply:
changed it to 1.1.0 on rust and go also